docs/submitting.md

```diff
@@ -54,10 +54,28 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
 
 While we are happy to review submissions in standalone repositories, we also review submissions that are significant contributions made to existing packages. It is often better to have an integrated library or package of methods than a large number of single-method packages.
 
-### Questions? Open an issue to ask
+### Questions? Open an issue to ask.
 
 Authors wishing to make a pre-submission enquiry should [open an issue](https://github.com/openjournals/joss/issues/new?title=Pre-submission%20enquiry) on the JOSS repository.
 
+## Conflict of Interest policy for authors
+
+Conflict of interest (COI) arises when an author has financial, personal, or other interests that may influence their research or the interpretation of its results. In order to maintain the integrity of the work published in JOSS, we require that authors disclose any potential conflicts of interest at submission time.
+
+### Policy
+
+**Disclosure:** All authors must disclose any potential conflicts of interest related to the research in their manuscript, including financial, personal, or professional relationships that may affect their objectivity. This includes any financial relationships, such as employment, consultancies, honoraria, stock ownership, or other financial interests that may be relevant to the research.
+
+**Acknowledgement:** Authors should acknowledge all sources of financial support for the research and include a statement indicating whether or not the sponsor had any involvement in the study design, data collection, analysis, or interpretation of the results.
+
+**Review process:** Editors and reviewers must be informed of any potential conflicts of interest before reviewing the manuscript to ensure unbiased evaluation of the research.
+
+**Mitigation of COI:** If a potential conflict of interest is identified, steps will be taken to mitigate the influence of the conflict, including but not limited to, disclosure in the manuscript and adjustment of the author's role in the research or removal of the author from the research entirely.
```

Comment:
This doesn't make sense - how could we adjust the author's role i the research or remove the author from the research?

Reply:
Yeah, sorry. I meant to edit this bit more before pushing the changes up. I'll remove this entirely.